_posts/2019-06-21-formats.md

```diff
@@ -0,0 +1,203 @@
+move to open formats or invest their own time or money to support their formats.
+
+<!-- TODO refactor caption.html -->
+<figure class="captioned">
+    <a href="https://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support/">
+        <img class="captioned"
+             src="{{ site.baseurl }}/images/2019/complex-formats.png"
+             alt="Statement about format complexity (2016)">
+    </a>
+    <figurecaption>Statement about format complexity
+    (<a href="https://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support/">2016</a>)</figurecaption>
+</figure>
+
+How did that turn out? Well, two years later the growth curve has naturally
+levelled off as we pursue other priorities. Currently there are just over 150
+formats supported. One company, 3i, has taken over support of their own file
+format
+([Slidebook6](https://www.intelligent-imaging.com/technical-answers#SLDinFIJI))
+with a closed source reader that lives outside of Bio-Formats.
+
+A few other companies have added support for their format either by
+contributing directly to the library or by commissioning Glencoe Software to
+do so. Where necessary the open source team has added support for formats
+that are needed for their funded priorities like datasets published in the
+Image Data Resource.[^1]<sup>,</sup>[^2]<sup>,</sup>[^3]<sup>,</sup>[^4]
+
+<blockquote>Paying for the initial cost of a format is not enough.</blockquote>
+
+But paying for the initial cost of a format is not enough. The need for
+indefinite support carries a larger, longer-lived price tag that leaves data
+written in a given format constantly at risk. These costs are exacerbated by
+format variants. Even when a format is defined following standards like DICOM,
+there is a need to contend with multiple implementations as is the case in the
+radiology domain. The same happened with the Olympus OIR format added in 2017
+in partnership with Olympus Europe. Following public release, the community has
+periodically reported breakages caused by new variants of the format.
+[^5]<sup>,</sup>[^6]<sup>,</sup>[^7]<sup>,</sup>[^8]<sup>,</sup>[^9]
+
+<blockquote>Put simply, the format landscape has scaled beyond a manageable level.</blockquote>
+
+Put simply, the format landscape has scaled beyond a manageable level. The
+result is that scientists end up blocked in accessing and properly handling
+their data, and thus blocked in their scientific endeavor. If Bio-Formats were
+to cease to exist, a large percentage of imaging data would immediately cease
+to be accessible at least until someone took on the burden of support.
+
+We understand the push to develop new formats. From numerous interactions, we
+know how critical both the speed at which data producers need to write as well
+as that at which users need to access their data are -- and that across as many
+platforms as possible. We also know that, optimally, this ecosystem should all
```

Comment:
> From numerous interactions, we know how critical both the speed at which data producers need to write as well as that at which users need to access their data are -- and that across as many
platforms as possible.

From numerous interactions, we know how crucial it is for data producers to be able to write and access their data quickly, and across as many platforms as possible.